CHANGELOG

```diff
@@ -2,6 +2,7 @@
 = ActiveMerchant CHANGELOG
 
 == HEAD
+* GlobalCollect: Provides more action-specific success Criteria for Global Collect's payment gatewa [poas14] #4308
```

Comment:
New entries go at the bottom of the HEAD list. Also, we usually wait to add these until the merge so we don't have ordering conflicts.

Reply:
ahh gotcha, so you just add it immediately before merging? will delete for now until merge